Algorithm.Python/TrainingExampleAlgorithm.py

```diff
@@ -0,0 +1,52 @@
+﻿# QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+# Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from clr import AddReference
+AddReference("System")
+AddReference("QuantConnect.Algorithm")
+AddReference("QuantConnect.Common")
+
+from System import *
+from QuantConnect import *
+from QuantConnect.Algorithm import *
+from time import sleep
+
+### <summary>
+### Example algorithm showing how to use QCAlgorithm.Train method
+### </summary>
+### <meta name="tag" content="using quantconnect" />
+### <meta name="tag" content="training" />
+class TrainingExampleAlgorithm(QCAlgorithm):
```

Comment:
Nice algorithm 👍
I think we could use _another_ algorithm as the C# versions that _tests_ the feature in python too